core/server/i18n.js

```diff
@@ -23,8 +36,14 @@ I18n = {
      * @returns {string}
      */
     t: function t(path, bindings) {
-        var string = I18n.findString(path),
-            msg;
+        var string, defaultString, msg;
+
+        currentLocale = I18n.locale();
+        if (bindings !== undefined) {
```

Comment:
Can you please add a test for showing the notation? Thanks!

Reply:
With the different fallbacks implemented, `defaultString` is now used only in rare situations, just as a last resort.

If for example we set `de` as locale, then the i18n system first tries to load the theme's _de.json_ translation file. If this file is missing, then it tries _en.json_, which is recommended in the help file to have as a minimum, especially in the particular case of using keys in the translation helpers. If even this is also missing, then the system uses `defaultString` from the default easy-to-read English text included -probably in most cases- in the translation helpers.

So, to test this last resort case, I've used translatable themes without those translation files, by renaming them, restarting Ghost, and setting different locales. Not sure how to automate this manual test, it sounds complex and very time consuming. Anyhow, the manual tests for `defaultString` are easy to do and have worked really well.